### PR TITLE
[READY] RCDs Can Now Eat Rods and Metal Floor Tiles For Matter Units

### PR DESCRIPTION
**Pull request by bobbah**
:cl: bobbahbrown
add: RCDs are now more environmentally friendly and can now recycle rods and floor tiles without having to reduce them to sheets first.
/:cl:

Old RCDs couldn't eat rods and metal floor tiles, it literally ruined my life.

Now I don't need to create sheets before I let the RCD harvest its bounty.

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
i ded

**Comment by bobbahbrown** [comment]
> i ded

sorry to hear that

File: code/game/objects/items/RCD.dm [hunk 1]
```diff
@@ -75,6 +75,10 @@ RLD
 		loaded = loadwithsheets(W, plasmarglassmultiplier*sheetmultiplier, user) //8 matter for one plasma rglass sheet
 	else if(istype(W, /obj/item/stack/sheet/rglass))
 		loaded = loadwithsheets(W, rglassmultiplier*sheetmultiplier, user) //6 matter for one rglass sheet
+	else if(istype(W, /obj/item/stack/rods))
+		loaded = loadwithsheets(W, sheetmultiplier * 0.5, user) // 2 matter for 1 rod, as 2 rods are produced from 1 metal
+	else if(istype(W, /obj/item/stack/tile/plasteel))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/RCD.dm]: metal floor tiles are typed as plasteel?

Comment (bobbahbrown) [reply code/game/objects/items/RCD.dm]: yes

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/RCD.dm]: whoever did this we are going to have a talk

Comment (bobbahbrown) [reply code/game/objects/items/RCD.dm]: > whoever did this we are going to have a talk

https://github.com/tgstation/tgstation/commit/867b21c7ce839999c89ae80929eff4aebdcf3d79#diff-4ec5a6d8b25e1125fc4adec16395c0aaL118

it's been at least six years

**Comment by Poojawa** [comment]
I remember when RCD ammo was a precious commodity.

**Comment by Barhandar** [comment]
It's still precious commodity because you get 16 charges per 4 sheets if you jam the whole sheet in, but 40 charges per 4 sheets if you print it in the upgraded lathe instead.

Also it takes hilariously retarded amounts of charges (floor is 33 for example) for removing things.

**Pull request merged**